bundles/org.openhab.binding.senechome/README.md

```diff
@@ -18,13 +18,14 @@ Examples: Lights, pool filters, wash machines, ...
 - only V3, V3duo have a power generator and thus MPPs (V3 has 2 MPP, V3duo has 3 MPP)
 - not equipped battery packs will return 0 for all ...Pack channels
 - currently channels for the first wallbox are implemented (senec could handle 4 wallboxes)
+- Senec disables http access at ~30.08.2023
 
 ## Thing Configuration
 
 demo.things
 
 ```java
-Thing senechome:senechome:pvbattery [ hostname="192.168.0.128", refreshInterval=60, limitationTresholdValue=70, limitationDuration=60 ]
+Thing senechome:senechome:pvbattery [ hostname="192.168.0.128", refreshInterval=60, limitationTresholdValue=70, limitationDuration=60, usePlain=false ]
```

Comment:
The new parameter is "useHttp" and not "usePlain".

Reply:
corrected